code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -557,6 +556,46 @@
 	do_sparks(3, TRUE, src)
 	..()
 
+///Disassembling the bot in a civilized manner with a multitool
+/mob/living/simple_animal/bot/medbot/disassemble()
+	var/turf/disassemble_turf = get_turf(src)
+
+	switch(skin)
```

Comment:
Is this duplicated from the switch in explode()? If so, can you maybe extract that behavior out to a common proc?

Reply:
Yes and I'll try to

Reply:
Done